src/content/docs/nrql/using-nrql/arrays-in-nrql.mdx

```diff
@@ -65,7 +61,7 @@ Simple arrays are used to store a list of values of the same primitive type; for
     className="freq-link"
     title={<InlineCode>contains(attribute, element)</InlineCode>}
   >
-    Use the `contains()` function to determine if an element is present in an array.
+    Use the `contains()` function to determine the presence of an element in an array.
```

Comment:
I actually prefer the previous version slightly. 

"determine" is the word here that's bogging this sentence down. How about "see"? "Use the ... function to see if there's an element in your array."